node/src/Transaction.ts

```diff
@@ -854,6 +855,21 @@ export class BaseTransaction<T extends BaseTransaction<T>> {
     public customCommand(args: string[]): T {
         return this.addAndReturn(createCustomCommand(args));
     }
+
+    /** Returns the element at index 'index' in the list stored at 'key'.
+     * The index is zero-based, so 0 means the first element, 1 the second element and so on.
+     * Negative indices can be used to designate elements starting at the tail of the list.
+     * Here, -1 means the last element, -2 means the penultimate and so forth.
+     * See https://redis.io/commands/lindex/ for more details.
+     *
+     * @param key - The sorted list.
+     * @param index - The index of the element in the list.
+     * @returns - The element at index in the list stored at key
```

Comment:
look at other transaction commands, instead of @returns we write `Command Response`